src/wallet/wallet.cpp

```diff
@@ -2959,6 +2959,18 @@ bool CWallet::CreateTransaction(interfaces::Chain::Lock& locked_chain, const std
                     return false;
                 }
 
+                // If the fee was capped to MAXTXFEE and this results in a too low fee rate given our current
+                // configuration, abort.
+                if (feeCalc.reason == FeeReason::MAXTXFEE) {
+                    if (nFeeNeeded < m_pay_tx_fee.GetFee(nBytes)) {
+                        strFailReason = _("Transaction is too large to fit in the maximum fee at the configured fixed fee rate");
+                        return false;
+                    } else if (nFeeNeeded < m_min_fee.GetFee(nBytes)) {
+                        strFailReason = _("Transaction is too large to fit in the maximum fee at the minimum fee rate");
```

Comment:
Is it me or this message is kind of confusing? Is it worth mentioning `fit in the maximum fee`?

Reply:
It is a tad confusing, but by removing the `...to fit in the maximum fee...` etc would make it collide with [this](https://github.com/bitcoin/bitcoin/blob/5f508cd31431010811954785c1a8708dd13e0402/src/wallet/wallet.cpp#L3085). It's the combination of size and fee parameters that makes it fail - not _just_ the size.

Reply:
I think I didn't explain well. I mean something as simple as `fee rate too low`?

Reply:
Changed to `Fee rate too low after limiting to -maxtxfee`. Just specifying `fee rate too low` could get people thinking _they_ configured it too low, which is not necessarily the case. Increasing the configured fee would make the problem worse.